examples/glyph-vector-2-cairo.py

```diff
@@ -0,0 +1,152 @@
+    bitmap = face.glyph.bitmap
+    width  = face.glyph.bitmap.width
+    rows   = face.glyph.bitmap.rows
+    pitch  = face.glyph.bitmap.pitch
+
+    Z = make_image_surface(bitmap)
+
+    outline = slot.outline
+    points = numpy.array(outline.points, dtype=[('x',float), ('y',float)])
+    x, y = points['x'], points['y']
+
+    bbox = outline.get_bbox()
+
+    MARGIN  = 10
+    scale = 3
+
+    width_s = ((bbox.xMax - bbox.xMin)//scale + 2 * MARGIN)
+    height_s = ((bbox.yMax - bbox.yMin)//scale + 2 * MARGIN)
+
+    surface = SVGSurface('glyph-vector-2-cairo.svg',
+                         width_s,
+                         height_s)
+    ctx = Context(surface)
+    ctx.set_source_rgb(1,1,1)
+    ctx.paint()
+    ctx.save()
+    ctx.scale(1.0/scale,1.0/scale)
+    ctx.translate(-bbox.xMin + MARGIN * scale,-bbox.yMin + MARGIN * scale)
+    ctx.transform(Matrix(1,0,0,-1))
+    ctx.translate(0, -(bbox.yMax + bbox.yMin)) # difference!
+
+    start, end = 0, 0
+
+    VERTS, CODES = [], []
+    # Iterate over each contour
+    for i in range(len(outline.contours)):
+        end    = outline.contours[i]
+        points = outline.points[start:end+1]
+        points.append(points[0])
+        tags   = outline.tags[start:end+1]
+        tags.append(tags[0])
+
+        segments = [ [points[0],], ]
+        for j in range(1, len(points) ):
+            segments[-1].append(points[j])
+            if tags[j] & (1 << 0) and j < (len(points)-1):
+                segments.append( [points[j],] )
+        verts = [points[0], ]
+        codes = [MOVETO,]
+        for segment in segments:
+            if len(segment) == 2:
+                verts.extend(segment[1:])
+                codes.extend([LINETO])
+            elif len(segment) == 3:
+                verts.extend(segment[1:])
+                codes.extend([CURVE3, CURVE3])
+            else:
+                verts.append(segment[1])
+                codes.append(CURVE3)
+                for i in range(1,len(segment)-2):
+                    A,B = segment[i], segment[i+1]
+                    C = ((A[0]+B[0])/2.0, (A[1]+B[1])/2.0)
+                    verts.extend([ C, B ])
+                    codes.extend([ CURVE3, CURVE3])
+                verts.append(segment[-1])
+                codes.append(CURVE3)
+        VERTS.extend(verts)
+        CODES.extend(codes)
+        start = end+1
+
+
+    # Draw glyph
+    ctx.new_path()
+    ctx.set_source_rgba(0.8,0.5,0.8, 1)
+    i = 0
+    while (i < len(CODES)):
+        if (CODES[i] == MOVETO):
+            ctx.move_to(VERTS[i][0],VERTS[i][1])
+            i += 1
+        elif (CODES[i] == LINETO):
+            ctx.line_to(VERTS[i][0],VERTS[i][1])
+            i += 1
+        elif (CODES[i] == CURVE3):
+            ctx.curve_to(VERTS[i][0],VERTS[i][1],
+                         VERTS[i+1][0],VERTS[i+1][1], # undocumented
+                         VERTS[i+1][0],VERTS[i+1][1])
+            i += 2
+        else:
+            raise NotImplementedError("Cannot cope!")
```

Comment:
I have some new code near that location to get 3rd order curves (what makes the latest figures at
https://github.com/rougier/freetype-py/issues/58 )